### PR TITLE
Move main plugin class to a PSR-4 file

### PR DESCRIPTION
**Pull request by szepeviktor**
@HPiirainen Noticed the strange array in PSR-4 autoloading.

### CODE REVIEW

File: src/Plugin.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+     * @var Importable[]
+     */
+    private static $importables = [];
+
+    /**
+     * Initialize the plugin.
+     */
+    public static function init() {
+        // Set the plugin version.
+        $plugin_data       = get_file_data( dirname( __DIR__ ) . '/plugin.php', [ 'Version' => 'Version' ], 'plugin' );
```
Comment (szepeviktor) [inline src/Plugin.php:56]: Here is the only change!

File: plugin.php
Comment (HPiirainen) [inline plugin.php:1]: Move the `use` statements from this class...

File: src/Plugin.php [hunk 2]
```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * The main plugin class.
+ */
+
+namespace Geniem\Oopi;
+
```
Comment (HPiirainen) [inline src/Plugin.php:7]: ... to this class so everything works correctly 👍

**Review by HPiirainen — CHANGES_REQUESTED** [review]
Thanks, good stuff! If you can move the `use` statements from the original `plugin.php` to the new `Plugin` class, then we're good to go 👍

File: plugin.php
Comment (szepeviktor) [reply plugin.php:1]: Done 🍏

File: src/Plugin.php (hunk 2 above)
Comment (szepeviktor) [reply src/Plugin.php:7]: Sorry. This is the first time I use a computer 🙃

**Pull request merged**